istio-telemetry/grafana/dashboards/istio-performance-dashboard.json

```diff
@@ -1373,7 +1373,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "process_virtual_memory_bytes{job=\"pilot\"}",
```

Comment:
I think I missed this on istio/istio, good catch. Hard to tell these issues from a huge json file